### PR TITLE
Clean up code, add more cpp11 features for performance

### PR DESCRIPTION
**Pull request by zmertens**
- Fix typos in global.xmh (addresses #19)
- Remove dead zeta.cpp file (addresses #19)
- Establish a constexpr army
- ~~Add noexcept to pre_walk in matic.cpp~~
- Convert NULL to nullptr

### CODE REVIEW

**Comment by reedacartwright** [comment]
Jenkins, okay to test.

File: src/lib/matic.cpp [hunk 1]
```diff
@@ -155,7 +155,7 @@ bool dawg::matic::finalize_configuration() {
 	return true;
 }
 
-void dawg::matic::pre_walk(alignment& aln) {
+void dawg::matic::pre_walk(alignment& aln) noexcept {
```
Comment (reedacartwright) [inline src/lib/matic.cpp]: I don't think noexcept is valid here.

**Comment by zmertens** [comment]
Update to remove noexceptness from pre-walk.

**Comment by reedacartwright** [comment]
Jenkins, okay to test

**Comment by reedacartwright** [comment]
Jenkins, retest this please.

**Pull request merged**